src/main/scala/com/fortysevendeg/android/sqlite/statement/SQLDroidStatement.scala

```diff
@@ -0,0 +1,196 @@
+package com.fortysevendeg.android.sqlite.statement
+
+import java.sql._
+
+import com.fortysevendeg.android.sqlite.logging.{AndroidLogWrapper, LogWrapper}
+import com.fortysevendeg.android.sqlite.resultset.SQLDroidResultSet
+import com.fortysevendeg.android.sqlite.{SQLDroidConnection, SQLDroidDatabase, WrapperNotSupported, _}
+
+import scala.collection.mutable
+
+class SQLDroidStatement(
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Number of methods in class exceeds 30](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/811801280/issues/source?bid=2442814&fileBranchId=2740117#l11)